src/api/async_element.rs

```diff
@@ -302,7 +275,7 @@ mod tests {
     }
 
     #[test]
-    fn one_async_element() {
+    fn one_async_element<'a>() {
```

Comment:
Necessary lifetime?

Reply:
Oh yeah, wasn't needed ,solved with Arc, will catch that on cleanup, also when I change the names and such.